### PR TITLE
fix/routing path

### PR DESCRIPTION
**Pull request by aarnphm**
- fix: routing should check if it start with /
- chore: wip
- chore: testing for path

## Description

prefix for path should be removed if users input via `route` options
```yaml
@svc.api(route="/api/v1/asdf")
```

## Checklist:

- [ ] My code follows the bentoml code style, both `make format` and
  `make lint` script have passed
  ([instructions](https://github.com/bentoml/BentoML/blob/master/DEVELOPMENT.md#style-check-and-auto-formatting-your-code)).
- [ ] My change reduces project test coverage and requires unit tests to be added
- [ ] I have added unit tests covering my code change
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

**Review by sauyon — APPROVED** [review]
LGTM!

**Comment by aarnphm** [comment]
I meed to update tests. Hold on

File: tests/e2e/bento_server_general_features/tests/test_service_api.py [hunk 1]
```diff
@@ -1,36 +1,84 @@
...
+    def r1(obj: str) -> int:
+        return 1
+
+    with pytest.raises(BentoMLException) as excinfo:
+
+        @svc.api(name="similar", input=Text(), output=Text())
+        def r2(obj: str) -> int:
+            return 1
+
+    assert "is already defined in Service" in str(excinfo.value)
```
Comment (bojiang) [inline tests/e2e/bento_server_general_features/tests/test_service_api.py:49]: nits: these are unit tests right?

Comment (aarnphm) [reply tests/e2e/bento_server_general_features/tests/test_service_api.py:49]: yes.

Comment (aarnphm) [reply tests/e2e/bento_server_general_features/tests/test_service_api.py:49]: I will refactor it out.

**Pull request merged**